test/functional/dive_test.go

```diff
@@ -527,12 +528,19 @@ var _ = ginkgo.Describe("DIVE CLI App", func() {
 	})
 
 	ginkgo.Describe("Icon chain commands", func() {
-		ginkgo.It("should run single icon node testing", func() {
+		ginkgo.It("should run single icon node", func() {
 			enclaveName := dive.GenerateRandomName()
 			cmd.Args = append(cmd.Args, "chain", "icon", "--enclaveName", enclaveName)
 			defer dive.Clean(enclaveName)
 			err := cmd.Run()
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			time.Sleep(5 * time.Second)
```

Comment:
Is 5 secs wait time really required ? It will slow down the execution time drastically can you kindly check and try to reduce wait time